tests/test-definitions.txt

```diff
@@ -41,7 +41,7 @@ shell_client_multi priority=2500 parallelity=2 single suffix=vst -- --vst true
 shell_client_transaction priority=750 single suffix=http -- $EncryptionAtRest
 shell_client_transaction priority=1750 single suffix=http2 -- --http2 true
 shell_client_transaction priority=2750 single suffix=vst -- --vst true
-shell_client_traffic priority=250 parallelity=1 single suffix=http -- http true
+shell_client_traffic priority=250 parallelity=1 single suffix=http
```

Comment:
both is wrong. it needs to be `-- --http true`. With this change the default protocol (which could be different) would be used.

Reply:
I know it should be `-- --http true` to explicitly make it use the HTTP protocol.
But I just looked at the other lines in the file (e.g. line 41 from the above excerpt), where we **DO NOT SET** the protocol explicitly when it is HTTP.
So for consistency I just did the same here.

Reply:
then all of the should be nailed to make this shure please.